x-pack/solutions/security/plugins/security_solution/server/lib/entity_analytics/privilege_monitoring/engine/initialisation_service.ts

```diff
@@ -111,61 +111,20 @@ export const createInitialisationService = (dataClient: PrivilegeMonitoringDataC
   const createOrUpdateDefaultDataSource = async (
     monitoringIndexSourceClient: MonitoringEntitySourceDescriptorClient
   ) => {
-    const sourceName = dataClient.index;
-
-    const defaultIndexSource: CreateMonitoringEntitySource = {
-      type: 'index',
-      managed: true,
-      indexPattern: defaultMonitoringUsersIndex(deps.namespace),
-      name: sourceName,
-    };
-
-    const existingSources = await monitoringIndexSourceClient.find({
-      name: sourceName,
-    });
-
-    if (existingSources.saved_objects.length > 0) {
-      dataClient.log('info', 'Default index source already exists, updating it.');
-      const existingSource = existingSources.saved_objects[0];
-      try {
-        await monitoringIndexSourceClient.update({
-          id: existingSource.id,
-          ...defaultIndexSource,
-        });
-      } catch (e) {
-        dataClient.log(
-          'error',
-          `Failed to update default index source for privilege monitoring: ${e.message}`
-        );
-        dataClient.audit(
-          PrivilegeMonitoringEngineActions.INIT,
-          EngineComponentResourceEnum.privmon_engine,
-          'Failed to update default index source for privilege monitoring',
-          e
-        );
-      }
-    } else {
-      dataClient.log('info', 'Creating default index source for privilege monitoring.');
-
-      try {
-        const indexSourceDescriptor = monitoringIndexSourceClient.create(defaultIndexSource);
-
-        dataClient.log(
-          'debug',
-          `Created index source for privilege monitoring: ${JSON.stringify(indexSourceDescriptor)}`
-        );
-      } catch (e) {
-        dataClient.log(
-          'error',
-          `Failed to create default index source for privilege monitoring: ${e.message}`
-        );
-        dataClient.audit(
-          PrivilegeMonitoringEngineActions.INIT,
-          EngineComponentResourceEnum.privmon_engine,
-          'Failed to create default index source for privilege monitoring',
-          e
-        );
-      }
+    dataClient.log('info', 'Creating default index source for privilege monitoring.');
```

Comment:
I believe we should put this behind a feature flag until the feature is finished